### ISSUE TITLE
--fastq_stats returns erroneous cumulated percentage for empty reads

### ISSUE THREAD

**Issue by frederic-mahe** [torognes/vsearch#571]
This is a minor bug. It can be reproduced like this:
```sh
printf "@s\n\n+\n\n" | \
    vsearch \
        --fastq_stats - \
        --log -
```
returns:
```
Read length distribution
      L           N      Pct   AccPct
-------  ----------  -------  -------
>=    0           1   100.0%   1844674407370955161600.0%
```
In the example above, the fraction of reads with this length zero or longer (`AccPct`) should be 100.0%. This seems to be caused by an out-of-bounds error: https://github.com/torognes/vsearch/blob/a267c0f6c24683d0d3e201348ee12de444b3f49e/src/fastqops.cc#L285

When `i == 0`, `length_dist[i - 1]` tries to read entry `-1` of the array `length_dist`.

**Comment by torognes** [comment]
Should be fixed in commit 2801e61.

**Comment by frederic-mahe** [comment]
I confirm that tests are now ok
